.github/workflows/build-plugin-archive.yml

```diff
@@ -258,7 +262,9 @@ jobs:
           ${{ inputs.PRE_SCRIPT }}
 
       - name: Run WP-CLI command
-        run: wp dist-archive . ./archive.zip --plugin-dirname=${{ steps.plugin-folder-name.outputs.plugin-folder-name }}
+        run: |
+          rm -rf interim-deps/.git
```

Comment:
I realize and respect that there likely isn't ever going to be a use-case for shipping the git clone in a build. 

I still wonder whether this should better be handled entirely by the `.distignore` file.